packages/workbox-google-analytics/src/initialize.ts

```diff
@@ -68,7 +68,7 @@ const createOnSyncCallback = (config: GoogleAnalyticsInitializeOptions) => {
         // Set the qt param prior to applying hitFilter or parameterOverrides.
         params.set('qt', String(queueTime));
 
-        // Apply `paramterOverrideds`, if set.
+        // Apply `parameterOverrideds`, if set.
```

Comment:
Oh man, looks like I had two typos in this one. Can you remove the last "d"?

Reply:
I took the liberty of fixing this so that we could close out the PR.